sys/shell/cmds/nanocoap_vfs.c

```diff
@@ -169,19 +169,34 @@ static int _nanocoap_put_handler(int argc, char **argv)
     file = argv[1];
     url = argv[2];
 
+    /* append filename to remote dir */
```

Comment:
Why? (I also find the difference between the comments wording "filename" & "remote dir" confusing, given that the variables are called "file" and "url", but that's very much nit picking.)